.gitignore

```diff
@@ -1,1071 +1,269 @@
-# Global ignores
-__pycache__/
+# Wildcards
+*.8
+*.a
+*.ao
+*.d
+*.dSYM
+*.db
+*.dll
+*.dylib
+*.exp
+*.lib
+*.lnk
+*.ncb
+*.o
+*.obj
+*.opendb
+*.opendf
+*.opensdf
+*.opt
+*.pc
+*.pdb
+*.plg
 *.pyc
+*.s
+*.sdf
+*.so
+*.suo
+*.tgz
+*.user
+*.vcproj.*.*.user
+*.vcxproj.user
+cygicudata*
+derb_*
+ht_*
```

Comment:
If it were up to me, I would make `derb_*` and `ht_*` directory-specific excludes